From ab1e5c6a72a00a7169ec719e87f524eaf5b26e36 Mon Sep 17 00:00:00 2001
From: Raphael Gallais-Pou <raphael.gallais-pou@foss.st.com>
Date: Wed, 13 Sep 2023 14:01:40 +0200
Subject: [PATCH] arm64: dts: st: add clock-cells to syscfg node on stm32mp251

Make the syscfg node a clock provider so clock consumers can reach child
clocks through device-tree.

Signed-off-by: Raphael Gallais-Pou <raphael.gallais-pou@foss.st.com>
Change-Id: I550c1188c8063595b05ab645a799696533e84727
---
 arch/arm64/boot/dts/st/stm32mp251.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/st/stm32mp251.dtsi b/arch/arm64/boot/dts/st/stm32mp251.dtsi
index d703783149a8..e5d546c7888c 100644
--- a/arch/arm64/boot/dts/st/stm32mp251.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp251.dtsi
@@ -2206,6 +2206,7 @@ exti-interrupt-map {
 		syscfg: syscon@44230000 {
 			compatible = "st,stm32mp25-syscfg", "syscon";
 			reg = <0x44230000 0x10000>;
+			#clock-cells = <1>;
 		};
 
 		exti2: interrupt-controller@46230000 {
-- 
2.39.5

